Makefile

```diff
@@ -9,7 +9,9 @@ CURRENT_GID := $(shell id -g)
 
 # git repo : equinix fabric-java sdk
 GIT_ORG=equinix-labs
-GIT_REPO=fabric-java
+GIT_REPO=fabric-sdk-java
+
+PACKAGE_VERSION=$(shell cat spec/oas3.fabric.config.json | jq -r '.artifactVersion')
```

Comment:
We need to remove `artifactVersion` and `artifactDescription` from `spec/oas3.fabric.config.json` file. They aren't being updated and they come directly from the OAS spec file.

Artifact version needs to come from a `version` file. See equivalent: https://github.com/equinix/equinix-sdk-go/blob/main/version

This gets read into PACKAGE_VERSION the following way: https://github.com/equinix/equinix-sdk-go/blob/c94de7bf1e1ffb039ad88ad7822a7518197a4b11/Makefile#L8C1-L8C37

We need to add a GHA job to update the `version` file. I believe the following job handles releases and updates to the version file: https://github.com/equinix/equinix-sdk-go/blob/main/.github/workflows/release.yaml